test/NamespaceTrackerSpec.hs

```diff
@@ -141,7 +143,7 @@ spec = do
                 ]
               })
             sendRev (i, Ocd $ ocdEmpty
-              { ocdDefinitions = Map.singleton helloTn helloDef1
+              { ocdDefinitions = Map.singleton (Tagged helloTn) helloDef1
```

Comment:
Why not define `helloTn` as being a tagged doodad of the right type instead of tagging at each use site.